test/cli-gateway-contract-completeness.test.ts

```
@@ -0,0 +1,227 @@
+  stableCommandNames,
+} from '../shared/cli-gateway-contract.js';
+import {
+  RELAY_COMMAND_MANIFEST,
+  relayCommandDefinitionsForSurface,
+} from '../shared/relay-command-manifest.js';
+
+/**
+ * Control character regex forbidding unescaped C0 control codes except \t (0x09), \n (0x0A), and \r (0x0D).
+ * Matches ASCII range [\x00-\x08\x0b\x0c\x0e-\x1f].
+ */
+const CONTROL_CHAR_REGEX = /[\x00-\x08\x0b\x0c\x0e-\x1f]/;
```

ESLint

[error] Unexpected control character(s) in regular expression: \x00, \x08, \x0b, \x0c, \x0e, \x1f.

(no-control-regex)